docs/en/01_usage.md

```diff
@@ -1,54 +1,68 @@
-# Using
+---
+title: TagField Module Usage
+summary: Managing tags on DataObjects, and integration with silverstripe-taxonomy
+icon: tags
+---
+
+# Usage
 
 The primary use, for this module, is as a custom input field interface. For instance, imagine you had the following data objects:
 
 ```php
+namespace App\Models;
+
 class BlogPost extends DataObject
 {
-	private static $many_many = array(
-		'BlogTags' => 'SilverStripe\\Blog\\Model\\BlogTag'
-	);
+    private static $many_many = [
+        'BlogTags' => BlogTag::class,
+    ];
 }
+```
+
+```php
+namespace App\Models;
 
 class BlogTag extends DataObject
 {
-	private static $db = array(
-		'Title' => 'Varchar(200)'
-	);
+    private static $db = [
+        'Title' => 'Varchar(200)',
+    ];
 
-	private static $belongs_many_many = array(
-		'BlogPosts' => 'SilverStripe\\Blog\\Model\\BlogPost'
-	);
+    private static $belongs_many_many = [
+        'BlogPosts' => BlogPost::class,
+    ];
 }
 ```
 
-If you wanted to link blog tags to blog posts, you might override `getCMSFields` with the following field:
+If you wanted to link blog tags to blog posts, you might override [`DataObject::getCMSFields()`](api:SilverStripe\ORM\DataObject::getCMSFields()) with the following field:
 
 ```php
 $field = TagField::create(
-	'BlogTags',
-	'Blog Tags',
-	BlogTag::get(),
-	$this->BlogTags()
+    'BlogTags',
+    'Blog Tags',
+    BlogTag::get(),
+    $this->BlogTags()
 )
-	->setShouldLazyLoad(true) // tags should be lazy loaded
-	->setCanCreate(true);     // new tag DataObjects can be created
+    // tags should be lazy loaded
+    ->setShouldLazyLoad(true)
+    // new tag DataObjects can be created
+    ->setCanCreate(true);
 ```
 
 **Note:** This assumes you have imported the namespaces class, e.g. `use SilverStripe\TagField\TagField;`.
```

Comment:
I was going to ask you to turn this into a callout block but it should just be removed, since the PHP example will have the appropriate `use` statements anyway.

Reply:
Removed